### ISSUE TITLE
Installation went perfectly, but GPU runtime will not work -- only CPU is used

### ISSUE THREAD

**Issue by MackNcD** [neonbjb/tortoise-tts#708]
To make ToirToiSe say the word "Hi." On the fastest setting, 1/2hr -- on 0% of my GPU is utilized, I can run the largest SD models whilst this inferencing is taking place. Any ideas on how to get the GPU involved XD

**Comment by MackNcD** [comment]
Just so you have confirmation that it's not taking advantage of the GPU, i got this error message upon the example given's test:
UserWarning: User provided device_type of 'cuda', but CUDA is not available. Disabling
I tried uninstalling 11.7 and putting 11.8 in there, i'm running everything through conda and nothing is missing aside from, well, GPU usage

**Comment by Artem111A** [comment]
I do not want to be that guy, but I have the same issue. I tried reinstalling the CUDA as well but it still didn't work.

**Comment by cgiles** [comment]
I did this : https://github.com/pytorch/pytorch/issues/30664#issuecomment-1707395814 and it fixed it for me

**Comment by Artem111A** [comment]
> I did this : [pytorch/pytorch#30664 (comment)](https://github.com/pytorch/pytorch/issues/30664#issuecomment-1707395814) and it fixed it for me

I just tried that and it still seems to be disabling CUDA.

**Comment by cgiles** [comment]
Does your cuda drivers are compatible with this version of pytorch ?

**Comment by MackNcD** [comment]
> Does your cuda drivers are compatible with this version of pytorch ?

Can't answer for Artem, but as for me, I've got everything up to 11.8 working for programs like SD, XTTSv2, etc.
I tried using docker for ToirToiSe and didn't have any luck there either. 
At this point I'm just going to try fine tuning XTTSv2 models, which isn't quite as good as toirtoise but it's got its advantages and perks (speed, and fine tuning allows you to get close)

**Comment by HandsomeDevilv112** [comment]
I'm having a similar problem. 
Things were working, then I decided to update my local directory.

**Comment by dabeckham** [comment]
Verify you have the cuda version of pytorch installed by  running the following commands at a python prompt, You should get a response TRUE. If you get FALSE you don't have pytorch installed properly.

import torch
torcn.cuda.is_available()

**Comment by GarbageHaus** [comment]
That's my go to as well for trouble-shooting. I tried updating this repo (The old one I have is from 8 months ago) but it was lacking some dependencies. Even now it looks like the repo doesn't contain the correct packages in the requirements.txt file. According to some random forum post, torch does not include cuda, so it may have been uninstalled when updating.

My last working configuration had the following:
torch                  1.11.0+cu113
torchaudio             0.11.0+cu113
torchvision            0.12.0+cu113

Maybe try installing those specific versions/packages? Kind of a shot in the dark.

**Comment by jurassicjordan** [comment]
was a solution found? I've been at it for a whole day now
